options/options.go

```diff
@@ -324,6 +324,7 @@ func (terragruntOptions *TerragruntOptions) Clone(terragruntConfigPath string) *
 		WorkingDir:                   workingDir,
 		Logger:                       util.CreateLogEntryWithWriter(terragruntOptions.ErrWriter, workingDir, terragruntOptions.LogLevel, terragruntOptions.Logger.Logger.Hooks),
 		LogLevel:                     terragruntOptions.LogLevel,
+		ValidateStrict:               terragruntOptions.ValidateStrict,
```

Comment:
I think will be helpful to have a unit test to keep track that in future required fields are copied

Reply:
I agree, but now the same problem is moved to a different place: if someone adds a new field to the struct and forgets to add it to the new unit test, then it doesn't help and may still be forgotten in `Clone`.  The only guarantee I can see here is some cleverness with the `reflect` package, which I'm not super keen on.

I'm assuming the unit test would be a simple "make a struct, call clone, modify the original, then assert that the clone has the original's old contents." Did you have something else in mind?

Reply:
I was thinking that may be implemented some kind of integration test, to verify if will fail `validate-inputs` execution 

More examples in:
https://github.com/gruntwork-io/terragrunt/blob/master/test/integration_test.go